Signal/src/ViewControllers/HomeView/Chat List/ChatListViewController.h

```diff
@@ -28,7 +28,9 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)unarchiveSelectedConversation;
 
 @property (nonatomic, readonly) CLVViewState *viewState;
-@property (nonatomic) TSThread *lastViewedThread;
+
+/// Used to update the selected cell for split view and maintain scroll positions for reappearing collapsed views.
+- (void)updateLastViewedThread:(TSThread *)thread animated:(BOOL)animated;
```

Comment:
Since `lastViewedThread` is only every accessed internally, I replaced the public property with a private property and a public update method.

Bonus fix: This property is always initially nil but the declaration marks it nonnull. Now the property is nullable, but its public update method still requires a nonnull thread argument.